livekit-agents/livekit/agents/stt/stream_adapter.py

```diff
@@ -24,7 +25,9 @@ def __init__(
         stt: STT,
         vad: VAD,
     ) -> None:
-        super().__init__(streaming_supported=True)
+        super().__init__(
+            capabilities=STTCapabilities(streaming=True, interim_results=False)
```

Comment:
should this reflect the capabilities of the thing it's wrapping? i.e. copy(stt.capabilities.copy) but update `.streaming=True`

Reply:
yeah makes sense. interim_results will always be False in the thing it's wrapping (I think) but anyways I think it's cleaner to do the copy. Will fix

Reply:
sg, yeah I guess it makes no difference today.. but might make a diff when more capabilities are added.

Reply:
the StreamAdapter *upgrades* the capability of a specific STT. 
If you want to get the real capabilities of the STT, you can use the wrapped_stt property and retrieve capabilities